playbooks/roles/miniconda/tasks/install_user.yml

```diff
@@ -1,10 +1,13 @@
 ---
-- name: Overwrite conda installation dir
```

Comment:
Using `set_fact` does not allow one to override a parameter passed to Ansible, since [the latter have higher precedence that host facts](https://stackoverflow.com/a/30134080). I therefore removed this task to allow us to override the `miniconda_install_dir` parameter in the SRC portal.